ui/src/publication/publication-card.tsx

```diff
@@ -36,11 +59,15 @@ const PublicationCard: React.FC<PublishListProps> = ({ publications, itemClicked
                     <h2 className={styles['publication-card__title']}>
                         {t('publication-card.title')}
                     </h2>
+
                     {failures.length > 0 && (
                         <React.Fragment>
                             <h3 className={styles['publication-card__subsection-title']}>
                                 {t('publishing.publish-issues')}
                             </h3>
+                            <p className={styles['publication-card__title-errors']}>
+                                {ratkoStatus && parseRatkoStatus(ratkoStatus)}
+                            </p>
```

Comment:
Tämän voisi kääntää niinpäin, että <p>-elementtiä ei näytetä ollenkaa, jos ratkoStatus on undefined